src/crypto/crypto.js

```diff
@@ -247,7 +247,7 @@ export function parseEncSessionKeyParams(algo, bytes) {
  * @param {Object}                  params  The key parameters indexed by name
  * @returns {Uint8Array}                    The array containing the MPIs
  */
-export function serializeKeyParams(algo, params) {
+export function serializeAlgorithmSpecificFields(algo, params) {
```

Comment:
Perhaps `serializeParams`? Bit less wordy.

Reply:
I was thinking about using `params`, I just thought this was more precise. Although, the objects have `param` keys rather than `AlgorithmSpecificFields` keys, so yeah, `params` would be consistent with that...

Changed it.